certbot/display/util.py

```diff
@@ -51,6 +50,21 @@ def _wrap_lines(msg):
 
     return os.linesep.join(fixed_l)
 
+
+def _clean(dialog_result):
+    """Work around inconsistent return codes from python-dialog.
```

Comment:
nit: The codes aren't really inconsistent. They're all [documented](http://pythondialog.sourceforge.net/doc/Dialog_class_overview.html), it's just we only use and expect a subset of them. Maybe this comment should reflect that more accurately?